src/core/server/server.api.md

```diff
@@ -1464,7 +1464,7 @@ export const kibanaResponseFactory: {
         message: string | Error;
         attributes?: ResponseErrorAttributes | undefined;
     }>;
-    customError: (options: CustomHttpResponseOptions<ResponseError>) => KibanaResponse<string | Error | {
+    customError: (options: CustomHttpResponseOptions<ResponseError | Buffer | Stream>) => KibanaResponse<string | Error | Buffer | Stream | {
```

Comment:
Looks ok to me, although as a nit, it would probably be good to add a note that explains the implications of using a Buffer or Stream as a `customError`

Reply:
> it would probably be good to add a note that explains the implications of using a Buffer or Stream as a customError

Can you add more details about what to add?